Directory.Packages.props

```diff
@@ -2,7 +2,7 @@
 <Project>
   <ItemGroup>
     <PackageVersion Include="BenchmarkDotNet" Version="0.13.12" />
-    <PackageVersion Include="BenchmarkDotNet.Diagnostics.Windows" Version="0.13.12" />    
+    <PackageVersion Include="BenchmarkDotNet.Diagnostics.Windows" Version="0.13.12" />
```

Comment:
doesn't look like this line needed to change right?

Reply:
It's removing the extra space after the element. It happened automatically with the update.